posts/inside-rust/2020-08-12-changes-to-x-py-defaults.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "Changes to x.py defaults"
```

Comment:
Do you want to show `x.py` as code like you did for `rustc`? For reference, that's also how it is in the [docs](https://rustc-dev-guide.rust-lang.org/building/how-to-build-and-run.html#what-is-xpy).

```suggestion
title: "Changes to `x.py` defaults"
```

Reply:
This is for the title page - I don't think browsers will render text as code in `<title>`, but I could be wrong.

Reply:
Ah, I meant more for the heading, but if this makes the title look weird then it's probably not worth it.

Reply:
Wow, it doesn't even render properly in the heading:

![image](https://user-images.githubusercontent.com/37223377/91354617-d8b96100-e7a1-11ea-8eab-145f54db9844.png)

Reply:
Yeah unfortunately it will just show up as backticks.
![image](https://user-images.githubusercontent.com/23638587/91354995-884efd00-e7bb-11ea-86d2-60c16b4dc8fc.png)